_includes/footer.html

```diff
@@ -1,7 +1,9 @@
 <footer class="footer">
+<style>.keylime-green {color:#40d837;}</style>
     <div class="container text-center">
+        <small class="copyright">Copyright the Keylime Contributors</small>
         <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can buy the commercial license via our website: themes.3rdwavemedia.com */-->
-        <small class="copyright">Designed with <i class="fas fa-heart"></i> by <a href="https://themes.3rdwavemedia.com" target="_blank">Xiaoying Riley</a> for developers</small>
+        <small class="copyright">Website template designed with <i class="fas fa-heart keylime-green"></i> by <a href="https://themes.3rdwavemedia.com" target="_blank">Xiaoying Riley</a> for developers</small>
```

Comment:
Since we're changing it, shouldn't it be "designed by"?

Reply:
I thought so at first, but his results in this:
>  Designed with ♥ by [Xiaoying Riley](https://themes.3rdwavemedia.com) for developers 

which makes it grammatically correct :)